packages/onboarding/src/utils/flows.ts

```diff
@@ -1,12 +1,18 @@
 export const NEWSLETTER_FLOW = 'newsletter';
+export const NEWSLETTER_POST_SETUP_FLOW = 'newsletter';
```

Comment:
should this be 'newsletter-post-setup'

Reply:
@Sharpirate - While this is technically a separate flow, I think we still want to treat it as part of the newsletter flow as far as this method is concerned. This is essentially like a sub-flow for newsletters.